components/rhs_thread/rhs_thread.test.tsx

```diff
@@ -133,4 +142,12 @@ describe('components/RhsThread', () => {
 
         expect(scrollToBottom).not.toHaveBeenCalled();
     });
+
+    test('should not break if root post is missing', () => {
+        (Utils.getRootPost as any).mockImplementation(() => undefined);
```

Comment:
I don't like the `(... as any)` here but vscode was showing a type violation. The tests succeed anyway. Let me know if you prefer it without the typecast.